examples/scenarioFlybyPoint.py

```diff
@@ -0,0 +1,338 @@
+
+    #
+    #  create the simulation process
+    #
+    dynProcess = scSim.CreateNewProcess(simProcessName)
+
+    # create the dynamics task and specify the integration update time
+    simulationTimeStep = macros.sec2nano(1)
+    dynProcess.addTask(scSim.CreateNewTask(simTaskName, simulationTimeStep))
+
+    #
+    #   setup the simulation tasks/objects
+    #
+
+    # initialize spacecraft object and set properties
+    scObject = spacecraft.Spacecraft()
+    scObject.ModelTag = "bsk-Sat"
+    # define the simulation inertia
+    I = [900., 0., 0.,
+         0., 800., 0.,
+         0., 0., 600.]
+    scObject.hub.mHub = 750.0  # kg - spacecraft mass
+    scObject.hub.r_BcB_B = [[0.0], [0.0], [0.0]]  # m - position vector of body-fixed point B relative to CM
+    scObject.hub.IHubPntBc_B = unitTestSupport.np2EigenMatrix3d(I)
+    scObject.hub.r_CN_NInit = [-5e7, 7.5e6, 0]  # m   - r_CN_N
+    scObject.hub.v_CN_NInit = [2e4, 0, 0]  # m/s - v_CN_N
+    # scObject.hub.sigma_BNInit = [[0.1], [0.2], [-0.3]]  # sigma_CN_B
+    # scObject.hub.omega_BN_BInit = [[0.001], [-0.01], [0.03]]  # rad/s - omega_CN_B
+
+    # add spacecraft object to the simulation process
+    scSim.AddModelToTask(simTaskName, scObject, None, 1)
+
+    #
+    # add RW devices
+    #
+    # Make a fresh RW factory instance, this is critical to run multiple times
+    rwFactory = simIncludeRW.rwFactory()
+
+    # create each RW by specifying the RW type, the spin axis gsHat, plus optional arguments
+    RW1 = rwFactory.create('Honeywell_HR16', [1, 0, 0], maxMomentum=50., Omega=100.)
+    RW2 = rwFactory.create('Honeywell_HR16', [0, 1, 0], maxMomentum=50., Omega=200.)
+    RW3 = rwFactory.create('Honeywell_HR16', [0, 0, 1], maxMomentum=50., Omega=300.)
+
+    numRW = rwFactory.getNumOfDevices()
+
+    # create RW object container and tie to spacecraft object
+    rwStateEffector = reactionWheelStateEffector.ReactionWheelStateEffector()
+    rwStateEffector.ModelTag = "RW_cluster"
+    rwFactory.addToSpacecraft(scObject.ModelTag, rwStateEffector, scObject)
+
+    # add RW object array to the simulation process.  This is required for the UpdateState() method
+    # to be called which logs the RW states
+    scSim.AddModelToTask(simTaskName, rwStateEffector, None, 2)
+
+    # add the simple Navigation sensor module.  This sets the SC attitude, rate, position
+    # velocity navigation message
+    sNavObject = simpleNav.SimpleNav()
+    sNavObject.ModelTag = "SimpleNavigation"
+    scSim.AddModelToTask(simTaskName, sNavObject)
+
+    #
+    #   setup the FSW algorithm tasks
+    #
+
+    # se tup flybyPoint guidance module
+    flybyGuid = flybyPoint.FlybyPoint()
+    flybyGuid.ModelTag = "flybyPoint"
+    flybyGuid.dtFilterData = 600
+    scSim.AddModelToTask(simTaskName, flybyGuid)
+
+    cameraAxis_B = [0, 1, 0]
+    outOfPlaneAxis_B = [1, 0, 0]
+    R0R = np.array([[0, -1,  0],
+                    [0,  0, -1],
+                    [1,  0,  0]])
```

Comment:
We should talk about this, there might be an error: if the camera boresight is in the y axis, I would expect the R0R tracking offset should be a +90 deg rotation about the Z axis.

Reply:
I believed this issue was resolved in the new version